src/website/views/templates/autopay.pug

```diff
@@ -0,0 +1,592 @@
+                  label.a-label(for='savings-routing-number') Routing number
+                  input(type="text", class="a-input -xl", id='savings-routing-number', placeholder="")
+              div.a-col.-w--12.-w-md--6
+                .m-formGroup.-labeled.-mb--2
+                  label.a-label(for='savings-account-number') Account number
+                  input(type="text", class="a-input -xl", id='savings-account-number', placeholder="")
+              div.a-col.-w--12
+                .-d--flex.-justify-content--center.-mt--2
+                  img.-mw--100.-h--auto(src='../../assets/images/autopay-check-hint.png', alt="savings account")
+      .a-card#creditDebitCardBase.-mt--1.-d--none
+        .a-card__header.-py--0.-bg--grey-10
+          h6 Add a credit or debit card
+        .a-card__content.-p--3
+          fieldset
+            .a-grid.-noGutter
+              div.a-col.-w--12
+                .m-formGroup.-labeled.-mb--2
+                  label.a-label(for='name') Name on card
+                  input(type="text", class="a-input -xl", id='name', placeholder="")
+              div.a-col.-w--12
+                .m-formGroup.-labeled.-mb--2
+                  div.-d--flex.-justify-content--between.-align-items--center
+                    label.a-label(for='number') Card number
+                    img(src='../../assets/images/autopay-credit-cards.jpg', alt="credit cards")
+                  input(type="text", class="a-input -xl", id='number', placeholder="")
+              div.a-col.-w-md--4.-w-sm--12
+                .m-formGroup.-labeled
+                  label.a-label(for='expiration') Expiration
+                  .-d--flex
+                    select(class="a-input -xl -mr--1", id='expiration')
+                      each day in days
+                        option= day
+                    select(class="a-input -xl -mr-md--1", id='expiration')
+                      each year in years
+                        option= year
+              div.a-col.-w-md--4.-w-sm--12
+                .m-formGroup.-labeled
+                  label.a-label(for='zip') Billing zip code
+                  input(type="text", class="a-input -xl", id='zip', placeholder="")
+  :code(lang="html")
+    <div class="chi-app -p3" id="autopayBase">
+      <div class="a-card -p3">
+        <div class="a-grid -noGutter">
+          <div><h5 class="-m--0 a-card__title">Automatic Payments</h5>
+            <p class="-text -mb--3">
+              Each month, an automatic payment for the full amount of your bill will be paid to
+              CenturyLink from the payment method of your choosing.
+            </p>
+          </div>
+          <div class="a-col -w-md--4">
+            <div class="m-formGroup -labeled -mb--1">
+              <label class="a-label" for="Base">Choose a payment method</label>
+              <div class="m-dropdown">
+                <button class="a-btn -xl m-dropdown__trigger" id="Base">Checking Account</button>
```

Comment:
Label casing is inconsistent. Change "Checking Account" to "Checking account"

Reply:
done